Sources/App/Models/AddAccessoryAirplaySupport.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  AddAccessoryAirplaySupport.swift
+//  App
+//
+//  Created by Kim de Vos on 27/07/2018.
+//
+
+import FluentPostgreSQL
+
+struct AddAccessoryAirplaySupport: PostgreSQLMigration {
```

Comment:
We could remove this class and manually add it. And then add `builder.field(for: \Accessory.supportsAirplay2)` in the `Accessory.prepare`

Reply:
Yeah, I think it's better if we do it manually on the DB only once and remove this class.

Reply:
I will do that!